scout/server/blueprints/variant/utils.py

```diff
@@ -0,0 +1,453 @@
+    
+    return frequencies
+
+
+def is_affected(variant_obj, case_obj):
+    """Adds information to show in view if sample is affected
+    
+    The views sometimes expect strings so we need to convert the raw data. This is an typical 
+    example of that situation
+    
+    Loop over the samples in a variant and add information from the case is they are affected
+    
+    Args:
+        variant_obj(scout.models.Variant)
+    """
+    individuals = {individual['individual_id']: individual for individual in
+                   case_obj['individuals']}
+    for sample_obj in variant_obj['samples']:
+        individual = individuals[sample_obj.get('sample_id')]
+        if not individual:
+            continue
+        sample_obj['is_affected'] = False 
+        
+        if individual['phenotype'] == 2:
+            sample_obj['is_affected'] = True 
+
+def evaluation(store, evaluation_obj):
+    """Fetch and fill-in evaluation object."""
+    evaluation_obj['institute'] = store.institute(evaluation_obj['institute_id'])
+    evaluation_obj['case'] = store.case(evaluation_obj['case_id'])
+    evaluation_obj['variant'] = store.variant(evaluation_obj['variant_specific'])
+    evaluation_obj['criteria'] = {criterion['term']: criterion for criterion in
+                                  evaluation_obj['criteria']}
+    evaluation_obj['classification'] = ACMG_COMPLETE_MAP.get(evaluation_obj['classification'])
+    return evaluation_obj
+
+def transcript_str(transcript_obj, gene_name=None):
+    """Generate amino acid change as a string.
+    
+    Args:
+        transcript_obj(dict)
+        gene_name(str)
+    
+    Returns:
+        change_str(str): A description of the transcript level change
+    """
+    # variant between genes
+    gene_part = 'intergenic'
+    part_count_raw = '0'
+
+    if transcript_obj.get('exon'):
+        gene_part = 'exon'
+        part_count_raw = transcript_obj['exon']
+    elif transcript_obj.get('intron'):
+        gene_part = 'intron'
+        part_count_raw = transcript_obj['intron']
+
+    part_count = part_count_raw.rpartition('/')[0]
+    change_str = "{}:{}{}:{}:{}".format(
+        transcript_obj.get('refseq_id', ''),
+        gene_part,
+        part_count,
+        transcript_obj.get('coding_sequence_name', 'NA'),
+        transcript_obj.get('protein_sequence_name', 'NA'),
+    )
+    if gene_name:
+        change_str = "{}:".format(gene_name) + change_str
+
+    return change_str
+
+
+def frequency(variant_obj):
+    """Returns a judgement on the overall frequency of the variant.
+
+    Combines multiple metrics into a single call.
+    
+    Args:
+        variant_obj(scout.models.Variant)
+    
+    Returns:
+        str in ['common','uncommon','rare']
+    """
+    most_common_frequency = max(variant_obj.get('thousand_genomes_frequency') or 0,
+                                variant_obj.get('exac_frequency') or 0,
+                                variant_obj.get('gnomad_frequency') or 0
```

Comment:
For the SVs I would include the largest available local one - gnomAD aint too good yet. I suppose that's clingen_mip currently.

Reply:
That would be new functionality, can we do that in another PR?